src/app/modules/customer-management/pages/customer.component.html

```diff
@@ -1,17 +1,22 @@
 <div class="container-fluid px-0 full-height">
   <div class="row">
     <div style="padding: 15px;">
-      <button (click)="activateCustomerForm()" class="btn btn-primary">Add new customer</button>
+      <button (click)="activateCustomerForm(); scrollToCustomerForm()" class="btn btn-primary">Add new customer</button>
     </div>
   </div>
   <app-customer-list
     [showCustomerForm]="showCustomerForm"
     [hasChange]="hasChangeComponent"
     (changeValueShowCustomerForm)="showCustomerForm = $event"
   ></app-customer-list>
-  <div class="row" *ngIf="showCustomerForm">
-    <div class="col">
-      <app-management-customer-projects (sendChanges)="getChangesInputs($event)" (closeCustemerForm)="closeCustomerForm($event)"></app-management-customer-projects>
+  <div id="bottom">
```

Comment:
Find a more representative id related to the component and not to its position.

Reply:
nice